includes/Core/Authentication/Authentication.php

```diff
@@ -1054,12 +1054,14 @@ private function get_reconnect_after_url_mismatch_notice() {
 				'content'         => function() {
 					$connected_url = $this->connected_proxy_url->get();
 					$current_url   = $this->context->get_canonical_home_url();
-					$content       = sprintf(
-						'<p>%s <a href="%s">%s</a></p>',
-						esc_html__( 'Site Kit by Google: Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.', 'google-site-kit' ),
+					$content       = '<p>' . sprintf(
+						/* translators: %1$s: Plugin name. %2$s: Notice content. %3$s: Proxy setup URL. %4$s: Reconnect string. */
+						__( '%1$s: %2$s <a href="%3$s">%4$s</a>', 'google-site-kit' ),
+						get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['Name'],
```

Comment:
On second thought, let's use an inline translation here – using `get_plugin_data` will translate everything, which is unnecessary. Should the plugin name ever change, it would be trivial to update other occurrences such as this. It's also a bit more clear what we're doing and makes it easier to find later if we go looking for it 😄 
```suggestion
						esc_html__( 'Site Kit by Google', 'google-site-kit' ),
```

Another benefit is that it makes it more obvious where escaping is needed.

Reply:
Sorted.